lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3389,7 +3389,11 @@ namespace {
                     ->getTopLevelModule()
                     ->getFullModuleName() == n;
         };
-        if (topLevelModuleEq(decl, "std")) {
+        if (topLevelModuleEq(decl, "std") ||
+            topLevelModuleEq(decl, "std_cmath") ||
```

Comment:
Why not use `starts with` here as well?

Reply:
No good reason, I’ll use `starts_with` here as well